lib/ahoy/intercom/tracker.rb

```diff
@@ -0,0 +1,26 @@
+require 'intercom'
+require 'ahoy/intercom/client'
+
+module Ahoy
+  module Intercom
+    module Tracker
+      include Client
+
+      def track(event_name, properties = {})
```

Comment:
well, the whole gem was supposed to be a store, not something to include in another store.